src/server.ts

```diff
@@ -271,6 +270,20 @@ export class Server {
     });
   }
 
+  /**
+   * Fetches metadata about the network which Soroban-RPC is connected to.
+   *
+   * @example
+   * server.getNetwork();
```

Comment:
lol

Reply:
I mean... I'll add some console.logs of the outputs for this example